SR2025/2025-01-08-jan-update.md

```diff
@@ -0,0 +1,62 @@
+---
+to: SR2025 teams
+subject: SR2025 January Update
+---
+
+Hi,
+
+Happy New Year! We hope you had a restful break and energised to continue building your robots!
+
+We have a few updates and reminders for you:
+
+* The next [Tech Days][tech-days] are in Cambridge, [on 25th January][cambridge-tech-day-january], and then back in Horsham [in February][horsham-tech-day-february] -- please [sign up][tech-day-signup] if you're interested
+* The second [Challenge submission deadline][challenge-deadline] is later this month
+* The [Virtual League][virtual-league] is coming up in February, and we have published a rules update clarifying how virtual league points are scaled
+
+## Second Challenge Deadline
+
+The [Challenges][challenges] offer stepping stones towards building your robot as well as allowing teams to collect valuable league points ahead of the competition. There are three Challenges which can be approached in any order.
+
+The second challenge deadline is coming up on Saturday **15th January 2025** at **18:00** UK time. Your team can submit any of the challenges by this time to earn league points for each they haven't already completed.
+
+There is one further submission deadline later in the year, though nothing stops your team getting all their submissions in sooner than that.
+
+## Virtual League
+
+Coming up in February, the [Virtual League][virtual-league] will be the first league matches of the competition. Run in the simulator, teams will pit their code against each other to score points. Teams must [submit their code][code-submitter-docs] by Thursday **6th February 2025** at **18:00** UK time and we'll be [livestreaming][virtual-league-livestream] the Virtual League from 1pm UK time on Saturday 8th.
```

Comment:
```suggestion
Coming up in February, the [Virtual League][virtual-league] will be the first league matches of the competition. Run in the simulator, teams will pit their code against each other to score points. Teams must [submit their code][code-submitter-docs] by Thursday **6th February 2025** at **6pm** UK time and we'll be [livestreaming][virtual-league-livestream] the Virtual League from 1pm UK time on Saturday 8th.
```
For consistency

Reply:
https://github.com/srobo/team-emails/pull/205/commits/2b11e32c7995ffed0f575a68aba1fe63b65ada10